### ISSUE TITLE
[BUG] (short description)

### ISSUE THREAD

**Issue by bryanhunwardsen** [MarlinFirmware/Marlin#19021]
### Bug Description

Marlin FW configured max feedrate is not being applied causing extruder skipping.

### My Configurations

I have replicated this on Ender3V2 with stock Creality Board running 2.0.6 release, I then swapped in an SKR mini E3 V2.0 with its stock Ender firmware and the problem remains. M203 shows an E value of 50, Sending G1 E150 F300 to the printer results in extruder skipping as the hotend cannot keep with that feedrate.

### Steps to Reproduce
1) Preheat to 210 for PLA
2) Send printer: G1 E150 F300
**Expected behavior:** [What you expect to happen]
1) FW limits extrusion to a Feed rate of 50 (or whatever is stored on the printer for E)
**Actual behavior:** [What actually happens]
Extrusion happens at 300 feedrate well over the maxfeedrate value set in Marlin
#### Additional Information
Not sure if this is in any way related to other open E3V2 axis/speed bug related to either long firmware names or print file names.

**Comment by sjasonsmith** [comment]
F300 is 300 mm/minute, so only 5mm/sec. That sounds slow, but is still about 12 cubic millimeters a second, which is probably more that your printer can sustain.

Have you actually measured the speed to see a mismatch, or are assuming this is a problem due to the skipped steps?

I suspect there is actually nothing wrong with the extrusion rate, and it is skipping because it can’t melt plastic fast enough. If you are actually seeing it move at 300 mm/s, then you have problems with other settings.

**Comment by sjasonsmith** [comment]
It is also worth mentioning that the default 50mm/s as far faster than any normal printer could extrude at. That speed is only useful for loading/unloading filament. You will usually limit your speed to a proper printing rate in the slicer.

I use the volumetric speed limit in PrusaSlicer, and I have to configure my Ender 3 down around 7 cubic mm/s to avoid problems. I get up in the 12+ range on another machine with a volcano hotend.

**Comment by AnHardt** [comment]
With a limit of `M203 E 50` in mm/second and a ordered feedrate 'G1 E150 F300' in mm/minute (= 5mm/second) you cant expect the feedrate to be limited. It's only 10% of the, with M203, allowed limit.

Please close.

**Comment by boelle** [comment]
no title and no config

and following the recommendation of @AnHardt

**Comment by bryanhunwardsen** [comment]
@AnHardt  Thanks for pointing out the unit differences, I was not aware and otherwise overlooked it while reading documentation during my troubleshooting. I was about to reference your comment and reopen the issue request with more verbose bug description/troubleshooting, observations, data and well as full configs when I realized that my Esteps where off on the SKR board (I had changed them but seems I forgot to save them and power cycled during testing). I corrected them nd retested and everything is working as expected now. (This confirms my true original issue that my Creality E3V2 v4 board went defective after two weeks overreporting temps so it was trying to print 10-20% lower than the reported temp. Is there a reason for the unit diferences between M203 and F, it seems otherwise problematic not not keep them in the same units? I ask because if there is no reason, I might consider doing a feature request to unify them or at least ask for M503 to report the current units rather than the not great "(units/s)" for feedrate. It might save future churn here for similar issue. 

@boelle Apologies sir, I have no clue how this issue request got double posted, the second one without a title(???)
